common/ta_errors.c

```diff
@@ -0,0 +1,158 @@
+#include "common/ta_errors.h"
+
+const char* ta_error_to_string(status_t err) {
+  switch (err) {
+    case SC_OK:
+      return "No error\n";
+    case SC_HTTP_OK:
+      return "HTTP response OK\n";
+    case SC_HTTP_BAD_REQUEST:
+      return "HTTP response, error when parsing request\n";
+    case SC_HTTP_NOT_FOUND:
+      return "HTTP request not found\n";
+    case SC_HTTP_INTERNAL_SERVICE_ERROR:
+      return "HTTP response, other errors in TA\n";
+    case SC_TA_OOM:
+      return "Failed to create TA object\n";
+    case SC_TA_NULL:
+      return "NULL TA objects\n";
+    case SC_TA_WRONG_REQUEST_OBJ:
+      return "Wrong TA request object\n";
+    case SC_TA_LOGGER_INIT_FAIL:
+      return "Failed to init ta logger\n";
+    case SC_TA_ERROR:
+      return "Failed to create cclient object\n";
```

Comment:
This status_t is `Failed in TA core function`
@HowJMay , this error code is only used in `accelerator/core/apis.c` when `ta_generate_address` fails. 
I think we should return the status_t directly instead of `SC_TA_ERROR`, so we can removed it.

Reply:
Sure. Good idea.

Reply:
@afcidk can you help us to remove it?